src/page/dashboard/__test__/DashboardPage.test.tsx

```
@@ -0,0 +1,223 @@
+    it('renders the dashboard with user name', () => {
+        render(<DashboardPage />);
+        expect(screen.getByText(/Welcome back/)).toBeInTheDocument();
+        expect(screen.getByText(/Test User!/)).toBeInTheDocument();
+    });
+
+    it('displays loading skeleton when credits are loading', () => {
+        vi.mocked(useCredits).mockReturnValue({
+            credits: 0,
+            orders: [],
+            isLoading: true
+        } as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)